src/sentry/api/endpoints/notification_defaults.py

```diff
@@ -10,7 +10,7 @@
 @control_silo_endpoint
 class NotificationDefaultsEndpoints(Endpoint):
     publish_status = {
-        "GET": ApiPublishStatus.PRIVATE,
+        "GET": ApiPublishStatus.EXPERIMENTAL,
```

Comment:
I will not merge this change. This is only to show these changes will be applied to the other repo. They are: https://github.com/getsentry/sentry-api-schema/blob/main/api_ownership_stats_dont_modify.json#L37